src/main/resources/db/migration/V092__admin_contact_categorisation.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE public.search_contact
+ADD COLUMN contact_type_id integer NULL;
+
+-- Set all contact type to be 'Enquiries' for now. Update when we have the confirmed list
+UPDATE public.search_contact
+SET contact_type_id = (SELECT id FROM public.admin_contacttype WHERE name = 'Enquiries');
```

Comment:
Guessing we won't merge this in until we get the list next week right? Just putting a note here to make sure we don't merge accidently until then :- )